x-pack/plugin/monitoring/src/main/java/org/elasticsearch/xpack/monitoring/exporter/local/LocalExporter.java

```diff
@@ -430,7 +430,7 @@ private boolean hasTemplate(final ClusterState clusterState, final String templa
 
     // FIXME this should use the IndexTemplateMetadataUpgrader
     private void putTemplate(String template, String source, ActionListener<AcknowledgedResponse> listener) {
-        logger.debug("installing template [{}]", template);
+        logger.warn("installing template [{}]", template);
```

Comment:
we probably don't want this change to go in?

Reply:
Indeed 👍